@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -530,7 +530,7 @@ export class TextElementsRenderer {
         // TODO: this is an intermeditate solution, in the end this
         // should not create a new cache, but update the former one
         this.m_textStyleCache = new TextStyleCache(textStyles, defaultTextStyle);
-        this.m_textStyleCache.initializeDefaultTextElementStyle(DEFAULT_FONT_CATALOG_NAME);
+        this.m_textStyleCache.initializeDefaultTextElementStyle();
```

Comment:
This change comes from Frauke's commit. Please rebase this PR on top of Frauke's merged change.